package.json

```diff
@@ -13,8 +13,7 @@
     "lint": "tslint --format stylish --project .",
     "lintfix": "tslint --fix --format stylish --project .",
     "build": "tsc",
-    "test": "mocha --compilers ts:ts-node/register test/",
-    "prepublish": "npm test"
```

Comment:
I'd like to see this remain, for the case of any manual publishing (eg publishing a prerelease or test branch) - at which point it should also include an `npm run build` I expect.  Maybe it's possible for circleci to skip the prepublish if it's already done it though?

Reply:
> Maybe it's possible for circleci to skip the prepublish if it's already done it though?

Best I can think of is some kind of `npm run manual-publish` which does `test && build && publish`, leaving circle free to just call `npm publish`.